oak-core-spi/src/main/java/org/apache/jackrabbit/oak/cache/CacheLIRS.java

```diff
@@ -1025,7 +1025,9 @@ V load(K key, int hash, Callable<? extends V> valueLoader) throws ExecutionExcep
             long start = System.nanoTime();
             try {
                 value = valueLoader.call();
-                loadSuccessCount++;
+                synchronized (this) {
+                    loadSuccessCount++;
```

Comment:
I think AtomicLong for all the counts would be even better... if we want to rely on the numbers